### PR TITLE
Put alerts at top of Route and MyBuses, Add a header for buses

### LINKED ISSUE #291: Move Alerts to the top of the routes page
If alerts are extremely important and can impact your trip, why scroll all the way down?

### PR DESCRIPTION
**Pull request by akaplo**
Closes #291. Also removes the "prerelease" bar at the top of MyBuses.

image

**__________________________________________________________________________________**

image

**__________________________________________________________________________________**

image

### CODE REVIEW

**Review by dfaulken — APPROVED** [review]
UI changes look solid - there is a merge conflict.

File: www/pages/my-buses/mybuses.html [hunk 1]
```diff
@@ -45,15 +50,6 @@
         <i class="icon ion-ios-plus-outline"></i>
         Add trip
       </a>
-      <div ng-if="messages.length > 0">
```
Comment (akaplo) [inline www/pages/my-buses/mybuses.html:48]: Entire block moved to line 3

File: www/pages/route/route.html [hunk 2]
```diff
@@ -18,32 +18,36 @@
...
+      Buses
+   </div>
     <div ng-if="vehicles && vehicles.length != 0">
       <ion-list>
         <div ng-repeat="vehicle in vehicles | orderBy:'Destination'">
           <vehicle data="vehicle" color="route.Color"></vehicle>
         </div>
      </ion-list>
    </div>
-     <!-- Container div for the alerts -->
```
Comment (akaplo) [inline www/pages/route/route.html:33]: Entire block moved to line 21

**Comment by akaplo** [comment]
Thanks @dfaulken. I'll merge, there's no code ambiguities here.

**Pull request merged**